.gitignore

```diff
@@ -1,5 +1,5 @@
 # Dependency directory
-node_modules
+# node_modules
```

Comment:
I think that as we are using `pack` command there is no need to commit `node_modules` folder, but you would need to start using `dist/index.js` file